### PR TITLE
Better IMEWindow

### PR DESCRIPTION
**Pull request by Loskh**
Moving the IME position to current cursor position.
ffxiv_dx11_g3qSTTcniv

### CODE REVIEW

File: Dalamud/Game/Gui/Internal/DalamudIME.cs [hunk 1]
```diff
@@ -73,6 +91,31 @@ internal void Enable()
                 this.interfaceHandle = Service<InterfaceManager>.Get().WindowHandlePtr;
                 this.wndProcPtr = Marshal.GetFunctionPointerForDelegate(this.wndProcDelegate);
                 this.oldWndProcPtr = SetWindowLongPtrW(this.interfaceHandle, WindowLongType.WndProc, this.wndProcPtr);
+
+                var module = Process.GetCurrentProcess().Modules.Cast<ProcessModule>().First(m => m.ModuleName == "cimgui.dll");
```
Comment (goaaats) [inline Dalamud/Game/Gui/Internal/DalamudIME.cs:95]: Did you consider sending a patch to expose this to upstream ImGui? I'm sure he'd be happy to merge/implement that, considering it's essential for a good IME implementation.

Not that I don't wanna merge it like this, but I'd be happier knowing that there was gonna be an official way to do it at some point.

Comment (Loskh) [reply Dalamud/Game/Gui/Internal/DalamudIME.cs:95]: okey, I'll try.

Comment (Loskh) [reply Dalamud/Game/Gui/Internal/DalamudIME.cs:95]: I found that **ImGui::GetCurrentContext()->PlatformImePos** is the pos of the cursor. But  ImGui.Net did not generate the struct  **ImGuiContext**.

Comment (goaaats) [reply Dalamud/Game/Gui/Internal/DalamudIME.cs:95]: @perchbird how annoying and terrible would it be to generate that? I think I need it in some other places too, would be cool to have

**Pull request merged**